README.md

```diff
@@ -25,16 +25,31 @@ The MyDRTV platform is designed to provide:
 ## Architectural Approach
 
 ### Architectural Styles Comparison
-We considered multiple architectural styles for the MyDRTV project. Below is a summary of the criteria used in our comparison:
+We considered multiple architectural styles for the MyDRTV project. Below is a summary of the criteria used in our comparison: 
 
 ## Technology Stack
 
-- **Frontend:** [Frontend technology, e.g., React, Angular]
-- **Backend:** [Backend technology, e.g., Node.js, Django]
-- **Database:** [Database, e.g., PostgreSQL, MongoDB]
-- **Search Engine:** [Search technology, e.g., Elasticsearch]
-- **Recommendation Engine:** [Algorithm or tech for recommendations]
-- **Infrastructure:** Cloud-hosted on [e.g., AWS, Azure] to ensure scalability and high availability.
+- **Frontend:** Typescript React. This choose is made over Javascript becuase we would like strongly typed language and gives better tooling then Javascript. Then React was choosen because it makes coding dynamic applications easier to write and more performant aswell as when we're already familiar with react beforehand.
```

Comment:
```suggestion
- **Frontend:** Typescript React. This choice was made over Javascript because we while we prefer a strongly typed language we also feel it provides better tooling than Javascript. React was chosen as our design library because of how dynamic it makes coding applications, because of our familiarity with the technology. Also out of the box, TypeScript supports JSX and which allows us to get the full React Web support that we want.
```